dashboards/kibana/createBackup.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# -E Set error traps to be inherited by function
+# -e Exits immediatly if a command exits with non-zero
+# -u Treat unset variables as an error when substituting(null is a valid value)
+# -o pipefail The return value of a pipeline is the status of the last command to exit with a non-zero status
+set -Eeuo pipefail
+
+## Constants
+
+KIBANA_API=https://kibana-assisted.apps.app-sre-prod-04.i5h0.p1.openshiftapps.com/_dashboards/api
```

Comment:
I wouldn't expose this in a public repo

Reply:
I'm fine removing that part but could you explain why ? That sounds like security through obscurity to me.

Reply:
I think there is a rule somewhere that public repos shouldn't expose internal implementation such as URLs.
However this rule is not respected everywhere. I agree that it's security through obscurity, but it's just a layer on top of other real security layers, so I think it's a reasonable request.